website_mass_mailing_name/static/src/js/website_mass_mailing_name.js

```diff
@@ -18,16 +18,26 @@ odoo.define("website_mass_mailing_name.subscribe", function (require) {
             return this._super(editable_mode);
         },
 
-        on_click: function() {
-            var email_error = !this.$email.val().match(/.+@.+/),
+        _onClick: function() {
+            function validateEmail(sEmail) {
```

Comment:
Sorry @victormmtorres I don't fully understand what you want to achieve here :slightly_smiling_face: You should not need to validate email here don't you? You already have that in the main method: https://github.com/odoo/odoo/blob/12.0/addons/website_mass_mailing/static/src/js/website_mass_mailing.js#L41-L64

Reply:
I will change this part